File: exer5.c
/**
 * Exercício 5, lista de exercícios 06-03-2013
 * Programa que calcula média ponderada de duas notas
 */

#include <stdio.h>

int main(int argc, char const *argv[])
{

	float nota1, peso1, nota2, peso2;

	printf("%s\n", "Programa que calcula média ponderada de duas notas:");
	printf("%s\n", "Digite a primeira nota:");
	scanf("%f", &nota1);

	printf("%s\n", "Ditite o peso da primeira nota:");
	scanf("%f", &peso1);

	printf("%s\n", "Digite a segunda nota:");
	scanf("%f", &nota2);

	printf("%s\n", "Digite o peso da segunda nota:");
	scanf("%f", &peso2);

	printf("%s %f\n", "A média ponderada entre estas duas notas é de:", (((nota1 * peso1) + (nota2 * peso2))/(peso1 + peso2)));

	return 0;
}

File: exer5.pas
Program CalMedPond;

Var
	nota1, peso1, nota2, peso2: real;

 Begin
	
	writeln('Programa para o cálculo da média ponderada entre dois valores');
	writeln('Escreva a primeira nota:');
	readln(nota1);
	
	writeln('Escreva o peso da segunda nota:');
	readln(peso1);
	
	writeln('Escreva a segunda nota:');
	readln(nota2);
	
	writeln('Escreva o peso da segunda nota:');
	readln(peso2);
	
	writeln('A média ponderada entre estes dois valores é:');
	write(((nota1 * peso1)+(nota2 * peso2))/(peso1 + peso2));

 End.